### PR TITLE
challenge completed and directory changed

### PR DESCRIPTION
**Pull request by Shivraj97**
Hello @gkanishk Please accept my PR.
I have completed all the 5 projects and 3 blogs before deadline.
Kindly add me to the finalist.

With Regards,
Shivraj Nag

### CODE REVIEW

File: content/finalist/ttjc@finalist@Shivraj97/index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: Shivraj97
+type: "finalist"
+name: "Shivraj Nag"
+portfolio: "https://shivrajnag.netlify.app/"
+github-link: "https://github.com/Shivraj97"
+linkedin-link: "https://www.linkedin.com/in/shivraj-n-5150a1b2/"
+looking-for: "job"
```
Comment (gkanishk) [inline content/finalist/ttjc@finalist@Shivraj97/index.md:8]: can you insert image link here, after line 8 as follows:

image-link: "link goes here"

Comment (Shivraj97) [reply content/finalist/ttjc@finalist@Shivraj97/index.md:8]: ok

**Comment by Shivraj97** [comment]
I have added image-link.
Please check.
Do I need to make any other changes...?

**Comment by gkanishk** [comment]
Awesome please share the required details inside #1033

**Pull request merged**